web/mattermost-ssl

```diff
@@ -1,34 +0,0 @@
-# server {
```

Comment:
I think we should keep ssl option. Agree that you want to use a ssl offloader, but at least this gives people an option to run the app in prod without any additional setup. E.g., for those who would like to do a small pilot, this would be an ideal option.